Tools/i18n/pygettext.py

```diff
@@ -475,46 +475,46 @@ def _extract_docstring(self, node):
 
     def _extract_message(self, node):
         func_name = self._get_func_name(node)
-        specs = self.options.keywords.get(func_name, [])
-        for spec in specs:
-            extracted = self._extract_message_with_spec(node, spec)
-            if extracted:
+        errors = []
+        for spec in self.options.keywords.get(func_name, []):
+            err = self._extract_message_with_spec(node, spec)
+            if err is None:
                 break
+            errors.append(err)
+        else:
+            for err in errors:
+                print(err, file=sys.stderr)
 
     def _extract_message_with_spec(self, node, spec):
         """Extract a gettext call with the given spec.
 
-        Return True if the gettext call was successfully extracted, False
-        otherwise.
+        Return `None` if the gettext call was successfully extracted,
+        otherwise return an error message.
         """
         max_index = max(spec.values())
         has_var_positional = any(isinstance(arg, ast.Starred) for
                                  arg in node.args[:max_index+1])
         if has_var_positional:
-            print(f'*** {self.filename}:{node.lineno}: Variable positional '
-                  f'arguments are not allowed in gettext calls', file=sys.stderr)
-            return False
+            return (f'*** {self.filename}:{node.lineno}: Variable positional '
+                   f'arguments are not allowed in gettext calls')
 
         if max_index >= len(node.args):
-            print(f'*** {self.filename}:{node.lineno}: Expected at least '
-                  f'{max_index + 1} positional argument(s) in gettext call, '
-                  f'got {len(node.args)}', file=sys.stderr)
-            return False
+            return (f'*** {self.filename}:{node.lineno}: Expected at least '
+                    f'{max_index + 1} positional argument(s) in gettext call, '
+                    f'got {len(node.args)}')
```

Comment:
With `--keyword=_:1 --keyword=_:2`, multiple errors will be printed for `_()`: "Expected at least 1 positional argument(s)...",  "Expected at least 2 positional argument(s)...".